src/custom-types/threshold.types.ts

```diff
@@ -7,6 +7,11 @@ export type UpperBoundValue = {
   inequalityMethod?: UpperBoundInequalityMethod;
 };
 
+export const UpperBoundInequalityMethods = {
```

Comment:
is this in the right file? looks like an implementation but in a types file. Needs cam-casing on the const declaration

Reply:
is not required anymore. will remove. switching  between different branches and clearly didnt clean up.